LanguageFeatures/Augmentation-libraries/type_inheritance_A01_t02_lib.dart

```diff
@@ -4,48 +4,48 @@
 
 /// @assertion An augmenting declaration may have no type annotations for a
 /// return type, variable type, parameter type, or type parameter bound type. In
-/// the last case, that includes omitting the extends keyword. For a variable or
-/// parameter, a var keyword may replace the type.
+/// the last case, that includes omitting the `extends` keyword. For a variable
+/// or parameter, a `var` keyword may replace the type.
 ///
 /// @description Check that augmenting declaration may have no type annotations
 /// for a variable type.
 /// @author sgrekhov22@gmail.com
 
-// SharedOptions=--enable-experiment=macros
+// SharedOptions=--enable-experiment=augmentations,enhanced-parts
 
 part of 'type_inheritance_A01_t02.dart';
 
 augment var topLevelVariable = "Augmented top-level variable";
-augment final finalTopLevelVariable = "Augmented top-level variable";
+augment final finalTopLevelVariable;
 
 augment class C {
   augment static var staticVariable = "Augmented static variable";
-  augment static final finalStaticVariable = "Augmented static variable";
+  augment static final finalStaticVariable;
   augment var instanceVariable = "Augmented instance variable";
-  augment final finalInstanceVariable = "Augmented instance variable";
+  augment final finalInstanceVariable;
 }
 
 augment mixin M {
   augment static var staticVariable = "Augmented static variable";
-  augment static final finalStaticVariable = "Augmented static variable";
+  augment static final finalStaticVariable;
   augment var instanceVariable = "Augmented instance variable";
-  augment final finalInstanceVariable = "Augmented instance variable";
+  augment final finalInstanceVariable;
 }
 
 augment enum E {
   augment e0;
```

Comment:
This would again be an error. But it should suffice to make it a lone `;` or adding a different value.